Scrapping_part_3/scrapping_part3/scrapping_part3/spiders/main.py

```diff
@@ -0,0 +1,202 @@
+import scrapy
+import time
+from requests import get
+
+
+# Create the class for our crawler
+class GpuDataScrapper (scrapy.Spider):
+
+    # Naming the crawler
+    name = "GPU_data_collector"
+
+    # Then we define the URLs of the websites we are going to scrape
+    start_urls = [
+        'https://www.regard.ru/catalog/group4000/page2.htm',
+        'https://www.onlinetrade.ru/catalogue/videokarty-c338/',
+        ]
+
+
+    REQUESTS_PER_SAME_TOR_IDENTITY = 100
```

Comment:
Why do you need this class attribute here? It should be configurable through settings.py module, not class attribute